openpathsampling/volume.py

```diff
@@ -359,8 +366,14 @@ def __sub__(self, other):
             return super(CVDefinedVolume, self).__sub__(other)
 
     def __call__(self, snapshot):
-        l = float(self.collectivevariable(snapshot))
-        return l >= self.lambda_min and l <= self.lambda_max
+        l = self.collectivevariable(snapshot).__float__()
+        if self.lambda_min != float('nan') and self.lambda_min > l:
+            return False
+
+        if self.lambda_max != float('nan') and self.lambda_max < l:
+            return False
+
```

Comment:
Not sure. It reported a string `nan` not having a `__float__` property so I just added that. Could be that it is actually only in the constructor and not the call test. Anyway I changed it to `inf` and `-inf`